examples/live/bybit/bybit_request_custom_endpoint.py

```diff
@@ -0,0 +1,148 @@
+from nautilus_trader.adapters.bybit.config import BybitDataClientConfig
+from nautilus_trader.adapters.bybit.config import BybitExecClientConfig
+from nautilus_trader.adapters.bybit.factories import BybitLiveDataClientFactory
+from nautilus_trader.adapters.bybit.factories import BybitLiveExecClientFactory
+from nautilus_trader.common import Environment
+from nautilus_trader.common.clock import TimeEvent
+from nautilus_trader.config import InstrumentProviderConfig
+from nautilus_trader.config import LiveExecEngineConfig
+from nautilus_trader.config import LoggingConfig
+from nautilus_trader.config import StrategyConfig
+from nautilus_trader.config import TradingNodeConfig
+from nautilus_trader.core.message import Request
+from nautilus_trader.core.nautilus_pyo3 import InstrumentId
+from nautilus_trader.core.uuid import UUID4
+from nautilus_trader.data.messages import DataResponse
+from nautilus_trader.live.node import TradingNode
+from nautilus_trader.trading import Strategy
+
+
+# *** THIS IS A TEST STRATEGY WITH NO ALPHA ADVANTAGE WHATSOEVER. ***
+# *** IT IS NOT INTENDED TO BE USED TO TRADE LIVE WITH REAL MONEY. ***
+
+# *** THIS INTEGRATION IS STILL UNDER CONSTRUCTION. ***
+# *** CONSIDER IT TO BE IN AN UNSTABLE BETA PHASE AND EXERCISE CAUTION. ***
+
+
+class RequestDemoStrategyConfig(StrategyConfig, frozen=True):
+    instrument_id: str
+    interval: int
+
+
+class RequestDemoStrategy(Strategy):
+    """
+    Strategy showcases how to request data from a custom bybit endpoint. Some adapter
+    endpoints are not connected to data engine, so they cannot be queried. We can use
+    the message bus to request data from these endpoints, as they are registered in
+    appropriate clients.
+
+    Also this strategy demonstrate:
+    - how to use the message bus to request data from a custom endpoint.
+    - how to use clock to schedule this request periodically.
+
+    """
+
+    def __init__(self, config: RequestDemoStrategyConfig):
+        super().__init__()
+        self.interval = config.interval
+        self.instrument_id = InstrumentId.from_str(config.instrument_id)
+        self.linear_request_ticker_uuid = UUID4()
+
+    def start(self):
+        seconds_delta = timedelta(seconds=self.interval)
+        self.clock.set_timer(
+            name="fetch_ticker",
+            interval=seconds_delta,
+            callback=self.send_tickers_request,
+        )
+
+    def send_tickers_request(self, time_event: TimeEvent):
+        request = Request(
+            request_id=self.linear_request_ticker_uuid,
+            ts_init=self.clock.timestamp_ns(),
+            callback=self.on_data,
+            metadata={"symbol": self.instrument_id.symbol},
+        )
+        self.msgbus.request(endpoint="bybit.data.tickers", request=request)
+
+    def on_data(self, data: DataResponse):
+        ## check generic data response by uuid
+        if data.correlation_id == self.linear_request_ticker_uuid:
```

Comment:
We can use some other method to know if this is a request ticker response.

Reply:
Pls check again the new implementation